apps/api/src/app/organization/usecases/create-organization/create-organization.usecase.ts

```diff
@@ -11,9 +11,7 @@ import { AddMemberCommand } from '../membership/add-member/add-member.command';
 import { AddMember } from '../membership/add-member/add-member.usecase';
 import { CreateOrganizationCommand } from './create-organization.command';
 
-@Injectable({
-  scope: Scope.REQUEST,
```

Comment:
is there any specific reason we're using request scope in the app? (there other providers in this PR that i update)
it’s currently limiting me from injecting certain providers into SystemOrganizationService, since it’s a singleton, and you can’t inject a request-scoped provider into a singleton.

cc @scopsy

Reply:
Yes, we need to be careful about this, when Scope is defined, we can be relying on this.someState to be only available on the request context. Vs shared with other requests. So you need to investigate the whole usecase tree to ensure we don't rely on this.somethingToKeepState

Reply:
good catch, i will check and refactor the usecase if needed.